lectures/eigen_II.md

```diff
@@ -411,7 +411,7 @@ Thus, the rate of convergence is governed by the modulus of the second largest e
 
 
 (la_neumann)=
-## The Neumann Series Lemma
+## The Neumann series lemma
```

Comment:
```suggestion
## The Neumann Series Lemma
```

Reply:
@HengchengZhang May I ask why we should capitalize all words here?

Reply:
Hi @DiaPorntipa, I think we are using all capitalized form from the start of this lecture, and it would be better to keep them consistent.

Reply:
Thanks @HengchengZhang. However, I am still confused. According to the manual, the heading above is the section heading, so I thought only the first word should be capitalized.

For reference,
<img width="1227" alt="Screenshot 2023-06-15 at 11 06 33 am" src="https://github.com/QuantEcon/lecture-python-intro/assets/71478665/be708b0b-7aea-4138-a7f1-ef47d4649f77">

Reply:
Thanks @DiaPorntipa for a serious look at this!

My opinion is that since it is a lemma itself and we use all capitalization in this lecture, it would be wried if we left this one with a different form.

Reply:
Thanks @HengchengZhang for the clarification. 

From a quick search, I found there are many variations regarding the capitalization of the lemma. Is that mean I should make changes to all of these words so that they are all capitalized?  
<img width="592" alt="Screenshot 2023-06-15 at 11 31 14 am" src="https://github.com/QuantEcon/lecture-python-intro/assets/71478665/d601ed57-4583-47a7-914e-abfd9cbbb37a">

Reply:
Hi @DiaPorntipa, that will be great!